### PR TITLE
Count Inversions in C

### PR DESCRIPTION
**Pull request by yr7000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description
Given an array of integers. Find the Inversion Count in the array.
Two elements a[i] and a[j] form an inversion if a[i] > a[j] and i < j.
Input: 
5
2 4 1 3 5
Output:
3

### Checklist

- [ yes] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [ yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#5166)

### CODE REVIEW

**Comment by yr7000** [comment]
Hey, @bhav09  please review this pr and provide some feedback.

File: C/sort/Count_Inversions.c [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    MergeSort(input, mid + 1, end);
+    Merge(input, start, end, mid);
+}
+
+// driver code.
+int main()
+{
+    // n is the size of the input array.
+    int n;
+    scanf("%d", &n);
```
Comment (ankitaggarwal23) [inline C/sort/Count_Inversions.c]: Remove the redundant comments, Use proper display message.

File: C/sort/Count_Inversions.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    int input[n];
+    for (int i = 0; i < n; i++)
+    {
+        scanf("%d", &input[i]);
```
Comment (ankitaggarwal23) [inline C/sort/Count_Inversions.c:67]: Use proper display message.

File: C/sort/Count_Inversions.c [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    }
+    MergeSort(input, 0, n - 1);
+    printf("%d", answer);
```
Comment (ankitaggarwal23) [inline C/sort/Count_Inversions.c]: Use proper display message.

File: C/sort/Count_Inversions.c [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+/*
+Time Complexity  : O(nlogn).
+Space Complexity : O(n).
+
+Input:
+5
+2 4 1 3 5
+Output:
+3
```
Comment (ankitaggarwal23) [inline C/sort/Count_Inversions.c]: Use proper display message.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C/sort/Count_Inversions.c (hunk 1 above)
Comment (yr7000) [reply C/sort/Count_Inversions.c:63]: sir @ankitaggarwal23, could you please explain the mistakes in detail. I am not understanding the meaning of the proper display message

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**